lib/protobuf/type_util.ex

```diff
@@ -36,7 +36,7 @@ defmodule Protobuf.TypeUtil do
   def enum_to_spec(:TYPE_SINT32), do: "integer"
   def enum_to_spec(:TYPE_SINT64), do: "integer"
   def enum_to_spec(_), do: "any"
-  def enum_to_spec(:TYPE_MESSAGE, type, true = _repeated), do: "#{type}.t"
+  def enum_to_spec(:TYPE_MESSAGE, type, true = _repeated), do: "[#{type}.t]"
```

Comment:
I just see that message type doesn't add `| nil` into generated type. Should we need to add into this patch?